akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -110,8 +113,7 @@ class Http2ServerSpec extends AkkaSpec("" + "akka.loglevel = debug")
           expectedRequest = HttpRequest(
             method = HttpMethods.GET,
             uri = "http://www.example.com/",
-            // FIXME: should be modeled header: headers.`Cache-Control`(CacheDirectives.`no-cache`()) :: Nil,
-            headers = RawHeader("cache-control", "no-cache") :: Nil,
+            headers = Vector(`Cache-Control`(`no-cache`)),
```

Comment:
Yes, modeled headers are uppercase, raw headers are lowercase. If you put uppercase in raw header, you will suffer. So maybe when HTTP/2 becomes dominant the modeled headers should go lower-case too. Not very important.